src/main/webapp/resources/js/pages/analysis/components/AnalysisMenu.jsx

```diff
@@ -0,0 +1,81 @@
+import { Menu } from "antd";
+import React, { useContext } from "react";
+import { Link, useLocation } from "react-router-dom";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { ANALYSIS } from "../routes";
+
+const { Item } = Menu;
+
+/**
+ * React component to render the menu for the analysis
+ * @returns {React.ReactElement}
+ */
+export default function AnalysisMenu({ type }) {
+  const location = useLocation();
+  const [current, setCurrent] = React.useState("");
+  const regex = /analysis\/\d+\/(?<path>\w+)/;
+  const { analysisContext, analysisIdentifier } = useContext(AnalysisContext);
+  const DEFAULT_URL = setBaseUrl(`/analysis/${analysisIdentifier}`);
+  const handleMenu = (e) => setCurrent(e.key);
+
+  React.useEffect(() => {
+    const found = location.pathname.match(regex);
+
+    if (found) {
+      setCurrent(found.groups.path);
+    } else {
+      if (type === "output") {
+        setCurrent("output");
+      } else {
+        setCurrent(type);
+      }
+    }
+  }, [location.pathname, analysisContext.isCompleted]);
+
+  return (
+    <Menu
+      selectedKeys={[current]}
+      mode="horizontal"
+      onClick={handleMenu}
+      className="t-analysis-menu"
+    >
+      {type === "sistr" && (
+        <Item key="sistr">
+          <Link to={DEFAULT_URL}>{i18n("Analysis.sistr")}</Link>
+        </Item>
+      )}
+      {type === "biohansel" && (
+        <Item key="biohansel">
+          <Link to={DEFAULT_URL}>{i18n("Analysis.biohansel")}</Link>
+        </Item>
+      )}
+      {type === "tree" && analysisContext.treeDefault && (
+        <Item key="tree">
+          <Link to={DEFAULT_URL}>{i18n("Analysis.phylogeneticTree")}</Link>
+        </Item>
+      )}
+      {type === "output" ? (
+        <Item key="output">
```

Comment:
I don't think we need an if here as it should still work to go to `${DEFAULT_URL}/${ANALYSIS.OUTPUT}`

Reply:
Updated in eea2103eb3e44a78e46de44d25a1b667fbee946c